console-subscriber/examples/deep.rs

```diff
@@ -0,0 +1,44 @@
+use tokio::task;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
+    use tracing_subscriber::prelude::*;
+
+    let subscriber = tracing_subscriber::registry()
+        .with(console_subscriber::spawn())
+        .with(tracing_subscriber::filter::filter_fn(|_| true));
```

Comment:
this could alternatively be `LevelFilter::TRACE` to enable everything explicitly; either is fine by me though.